app/assets/stylesheets/_shared.scss

```diff
@@ -23,3 +23,7 @@ body {
 .site-footer {
   background-color: $color-light-blue;
 }
+
+.btn-success {
+  text-shadow: 0 1px 1px rgba(0, 0, 0, 0.3);
```

Comment:
Color literals like `rgba(0, 0, 0, 0.3)` should only be used in variable declarations; they should be referred to via variable everywhere else.